src/app/lib/deleted_document.coffee

```diff
@@ -0,0 +1,40 @@
+log = require('./persistent_log')
+    prefix: "DeletedDocument"
+    date: true
+instance = null
+
+
+###*
+ * ChangeFileHandler allows to download, rename and delete a file.
+ *
+ * @class ChangeFileHandler
+###
+module.exports = class DeletedDocument
+
+
+    ###*
+     * Create a ChangeFileHandler.
+    ###
+    constructor: ->
+        return instance if instance
+        instance = @
+
+        @remoteDb = app.init.database.remoteDb
+
+
+    get: (docId, callback) ->
+        options =
+            revs: true
+            open_revs: 'all'
+        @remoteDb.get docId, options, (err, response) =>
+            return callback err if err
+
+            if response.length > 0
+                doc = response[0].ok
+                return callback new Error "" unless doc
+
+                revision = parseInt(doc._rev[0], 10) - 1
+                revision += "-" + doc._revisions.ids[1]
```

Comment:
You are sure it is `[1]`? I've done things more complicated than that for cozy-desktop, but I don't remember why...

Cf https://github.com/cozy-labs/cozy-desktop/blob/master/src/pouch.coffee#L178-L194